### DISCUSSION TITLE
Complex gradient

### DISCUSSION THREAD

**Discussion by NATateo** [cduck/drawSvg#130] [Q&A]
Hi and thank you for this great library

I need to draw a full color wheel, i.e. a circle filled with the entire color spectrum like this:
![wheel-5-ryb](https://github.com/cduck/drawsvg/assets/170121725/88bff56c-04d3-4b04-a8f8-2db2085f279e)

What's the best way of doing it?

**Comment by cduck** [comment]
Good question.  I don't know a good way to do this in SVG.  These resources might help: https://stackoverflow.com/questions/18206361/svg-multiple-color-on-circle-stroke/18210763#18210763, https://www.w3.org/Graphics/SVG/WG/wiki/Proposals/Advanced_Gradients
